Gopkg.toml

```diff
@@ -119,4 +119,4 @@ ignored = ["google.golang.org/appengine*"]
 
 [[constraint]]
   name = "github.com/prometheus/client_golang"
-  version = "0.9.0"
+  version = "0.9.0"
```

Comment:
This probably should have a new line at the end of this file.

Reply:
done.